packages/truffle-migrate/migration.js

```diff
@@ -92,7 +92,7 @@ class Migration {
       if (options.save === false) return;
 
       // Write migrations record to chain
-      const Migrations = resolver.require("./Migrations.sol");
+      const Migrations = resolver.require("Migrations.sol");
```

Comment:
We could even go further and just use the contract name (contract name is the correct form for `artifacts.require`)

```suggestion
      const Migrations = resolver.require("Migrations");
```